src/modality-agnostic-files/provenance.md

```diff
@@ -0,0 +1,738 @@
+# Provenance
+
+Support for provenance was developed as a [BIDS Extension Proposal](../extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../introduction.md#citing-bids) on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+!!! bug
+    Change example links below once they are merged to bids-examples
+
+!!! example "Example datasets"
+
+    The following examples have been formatted using this specification
+    and can be used for practical guidance when curating a new dataset.
+
+    - [Provenance of DICOM to Nifti conversion with `dcm2niix`](https://github.com/bclenet/bids-examples/tree/BEP028_dcm2niix/provenance_dcm2niix) - [Associated Pull Request #494](https://github.com/bids-standard/bids-examples/pull/494)
+    - [Provenance of DICOM to Nifti conversion with `heudiconv`](https://github.com/bclenet/bids-examples/tree/BEP028_heudiconv/provenance_heudiconv) - [Associated Pull Request #496](https://github.com/bids-standard/bids-examples/pull/496)
+    - [Provenance of fMRI preprocessing with `SPM`](https://github.com/bclenet/bids-examples/tree/BEP028_spm/provenance_spm) - [Associated Pull Request #497](https://github.com/bids-standard/bids-examples/pull/497)
+    - [Provenance of fMRI preprocessing with `fMRIPrep`](https://github.com/bclenet/bids-examples/tree/BEP028_fmriprep/provenance_fmriprep) - [Associated Pull Request#502](https://github.com/bids-standard/bids-examples/pull/502)
+
+    Further datasets are available from
+    the [BIDS examples repository](https://bids-website.readthedocs.io/en/latest/datasets/examples.html#provenance).
+
+## Overview
+
+This part of the BIDS specification is aimed at describing the provenance of a BIDS dataset.
+
+This description is retrospective: it describes a set of steps that were executed in order to obtain the dataset.
+
+!!! note
+    This is different from prospective provenance that focuses describing workflows that may be run on a dataset.
+
+This description is based on the [W3C Prov](https://www.w3.org/TR/2013/REC-prov-o-20130430/) standard.
+
+### General principles
```

Comment:
These principles could be moved into the top level section, other sections of the standard (behavioral, phenotypic, etc) use requirement keywords in their opening salvos. I'd also remove the new lines between the sentences.